tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -173,8 +182,37 @@ def test_brick_evict_with_node_down(self):
                 "Failed to bring down node {}".format(hostname))
 
         # Perform brick evict operation
-        try:
```

Comment:
This can be simplified as -
```
        try:
            heketi_ops.heketi_brick_evict(h_node, h_server, brick_id)
        except AssertionError as e:
            if (node_count == '' and 'No Replacement was found' not in six.text_type(e)):
            	raise
```

Reply:
node_count == '' and 'No Replacement was found' not in six.text_type(e) These two conditions are contradictory and will never meet.

Reply:
then `node_count == 3' and 'No Replacement was found' not in six.text_type(e)'` will work ?

Reply:
It seems to be working fine. Added this now.